src/commands/command-fns/blog.js

```diff
@@ -0,0 +1,85 @@
+const got = require('got')
+const {getCommandArgs} = require('../utils')
+
+async function fetchArticles() {
+  const response = await got('https://kentcdodds.com/blog.json')
+
+  return JSON.parse(response.body)
+}
+
+function printArticles(articles) {
+  return articles
+    .map(article => {
+      return `- ${article.title}
+  <${article.productionUrl}>`
+    })
+    .join('\n')
+}
+
+function searchArticles(articles, searchText) {
+  const regex = new RegExp(searchText, 'ig')
+  return articles.filter(
+    article =>
+      article.title.match(regex) ||
+      article.description.match(regex) ||
+      article.categories.find(category => category.match(regex)) ||
+      article.keywords.find(keyword => keyword.match(regex)),
+  )
+}
+
+async function blog(message) {
+  const args = getCommandArgs(message.content).trim()
+  let articles
+
+  try {
+    articles = await fetchArticles()
+  } catch (error) {
+    return message.channel.send(
+      `🤯 Something went wront retrieving the list of articles. Could you try in a few minutes?😀?`,
+    )
+  }
+
+  if (args === 'last') {
+    const lastArticles = articles.slice(0, 10)
+    message.channel.send(`This is the list of the last 10 articles on the blog:
+${printArticles(lastArticles)}`)
```

Comment:
If we go on multiple lines, then I'd prefer we commit to it 😅

```suggestion
    message.channel.send(
      `
This is the list of the last 10 articles on the blog:
${printArticles(lastArticles)}
      `.trim(),
    )
```